docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -156,61 +174,86 @@ responsible-roles:
   - Customer
 x-trestle-set-params:
   ac-1_prm_1:
-    values: all personnel
-    profile-values: new value from profile
+    label: label from edit
+    profile-values: all personnel
+    values: Param_1_value_in_catalog
+    display-name: Pretty ac-1 prm 1
+    ns: https://display-namespace
```

Comment:
"ns" of diplay-name property is not being shown as it is a trestle namespace prop.

Reply:
removed - along with an old reference to default-namespace that no longer applies.